test/active_support_test_case_helper.rb

```diff
@@ -131,7 +131,7 @@ def setup_user(operation, type = "", search = nil, user = :one)
     @one = users(user)
     as_admin do
       permission = Permission.find_by_name("#{operation}_#{type}") ||
```

Comment:
If the permission doesn't have to be in the database, is finding it in the database even needed then?